include/mrdox/Plugin.hpp

```diff
@@ -0,0 +1,45 @@
+//
+// This is a derivative work. originally part of the LLVM Project.
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// Copyright (c) 2023 Vinnie Falco (vinnie.falco@gmail.com)
```

Comment:
Klemens

Reply:
hey it's my copyright, I am free to give it away.